.github/workflows/durations.yml

```diff
@@ -0,0 +1,53 @@
+name: Update Durations
+
+on:
+  # Every Sunday at 00:00 UTC
+  # https://crontab.guru/#0_0_*_*_0
+  schedule:
+    - cron: '0 0 * * 0'
+
+  workflow_dispatch:
+
+jobs:
+  update-durations:
+    runs-on: ubuntu-latest
+    steps:
+      - name: download recent artifacts
+        run: >
+          gh run list
+            --branch main
+            --workflow tests
+            --limit 10
+            --json databaseId
+            --jq '.[].databaseId'
+          | xargs
+            -n 1
+          gh run download
+            --dir ${{ runner.temp }}/artifacts/
+            --pattern '*-all'
+
+      - uses: actions/setup-python@v4
+        with:
+          python-version: '3.11'
+
+      - uses: actions/checkout@v3
+
+      - name: combine recent durations from artifacts
+        run: python tests/durations/combine.py ${{ runner.temp }}/artifacts/
+
+      - name: create updated durations PR
+        permissions:
+          contents: write
+          pull-requests: write
+        uses: peter-evans/create-pull-request@v5
+        with:
+          push-to-fork: conda-bot/conda
+          token: ${{ secretes.DURATIONS_TOKEN }}
```

Comment:
Need to create this secret before this workflow will work

Reply:
Do you need help with this?

Reply:
I'm actually thinking to update this to not use a fork and to use elevated permissions instead of defining a new token